### PR TITLE
add bio page for Hour of Code review committee

### PR DESCRIPTION
**Pull request by tanyaparker**
image

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/review-committee.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Hour of Code and CS Ed Week Review Committee
+---
+# Hour of Code and CS Ed Week Review Committee
+
+<%= view :about_headshots, people:DB[:cdo_team].where(kind_s:'hoc_review'), :columns=>3 %>
+
+<%= view :about_people, people:DB[:cdo_team].where(kind_s:'hoc_review_short') %>
```
Comment (caleybrock) [inline pegasus/sites.v3/hourofcode.com/public/review-committee.md:8]: so you don't have any of these short ones right now right?

**Comment by caleybrock** [comment]
For Darlene / others without a headshot, we should have a backup case because this looks a bit weird, like it's still trying to load. Maybe a default gray circle or icon would be better?

File: pegasus/sites.v3/hourofcode.com/public/review-committee.md (hunk 1 above)
Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/public/review-committee.md:8]: Nope, but there may be in the future.

**Comment by tanyaparker** [comment]
Asked Maddie to ping her again if she can submit a picture. I'll try both a gray picture and icon. Do you like [user](http://fontawesome.io/icon/user/)?

**Comment by caleybrock** [comment]
I like user, but I think it should at least be in a circle just so it looks consistent.

**Comment by tanyaparker** [comment]
Something like this?

image

**Comment by caleybrock** [comment]
👍 looks better, thanks!

**Pull request merged**